gitbook/first-steps/basispakete.md

```diff
@@ -5,7 +5,7 @@ Um die `GeoStyler` dependency hinzuzufügen, navigieren Sie bitte (falls noch ni
 Ihres Projektes und führen Sie dort folgenden Befehl aus
 
 ```javascript
-npm install geostyler
+npm install geostyler@721
```

Comment:
Are you not missing the dots between the the different parts of the semver version?